specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/ManagedPrivateEndpoint.json

```diff
@@ -54,7 +54,10 @@
       },
       "additionalProperties": {
         "type": "object"
-      }
+      },
+      "required": [
+        "type"
+      ]
```

Comment:
We are setting `type` as required here, but I did not find a property in this model with name of `type`. How could this happen?

Reply:
In addition, this model has marked `type` as discriminator, which requires us to make the property `type` required. But still, I did not find this property in this model. Therefore we might need more fix to make this swagger reasonable

Reply:
the change is the advice of @lmazuel . And I also sent e-mail called 'RE: AutoRest error about azure-sdk-for-python-track2'. Do you think we need make a quick meeting to discuss about it?